source/clear-linux/guides/maintenance/mixin.rst

```diff
@@ -56,35 +56,52 @@ Copy your custom RPM package to the workspace
    Follow the instructions on how to build RPMs found at the
    `Developer tooling framework for Clear Linux`_.  
 
-Copy your RPM package to the workspace.
+If you have a local RPM you want to add to your mix you can do so by copying
+your RPM package to the workspace.
 
 .. code-block:: console
 
    $ sudo cp [RPM] /usr/share/mix/rpms
 
+Alternatively, you can add a remote RPM repository by running the following
+command.
+
+.. code-block:: console
+
+   $ sudo mixin repo add [repo-name] [repo-url]
+
 Create a bundle with your custom RPM package
 ********************************************
 
-Use the :command:`swupd-add-pkg` command to create a bundle with the RPM
+Use the :command:`mixin` command to create a bundle with the RPM
 package.
 
 .. code-block:: console
 
-   $ sudo swupd-add-pkg [RPM] [bundle-name]
+   $ sudo mixin package add [package-name] [--build]
+
+This command will add package-name to a bundle named after the repository it
+came from. For example, if the RPM was provided locally, it will be added to the
```

Comment:
Can we reword this sentence? Do either of these match your intent?  

- This command adds a package-name to a bundle, after which it's named. 
- This command adds a package-name, named after its repository, to a bundle.

Reply:
@matthewrsj If there's potential for a misreading, I want to offer options to clarify. 

This command will add a package-name to a bundle that is named after its parent repository.

Reply:
Ah, yes that's more clear. I'll change the sentence.